wiki/fr/fr-Onboarding.md

```diff
@@ -0,0 +1,58 @@
+
+### 1. Configurez un profil
+Pour que les autres sachent qui vous êtes, vous devez configurer votre *profil*. Pour le faire, cliquez sur « Vue » (barre des menus) > « Mon profil… ».  
+Maintenant vous allez voir :
+
+![Profil de musicien](https://user-images.githubusercontent.com/9108457/101895791-48fcbe80-3ba8-11eb-913e-d0e4e7d60666.png)
+
+Remplissez au minimum votre « Pseudo/nom » pour renseigner votre nom et fermez la fenêtre. 
+
+### 2. Connectez vous à un serveur
+Maintenant que vous êtes configuré, vous pouvez utiliser le bouton « Se connecter » dans la fenêtre principale de Jamulus pour rejoindre un serveur. Une fenêtre s'ouvre alors dans laquelle vous voyez les *serveurs regroupés par genre*. Pour choisir un autre genre, sélectionnez-en un dans la liste en haut à gauche. Après avoir sélectionné un genre et un serveur, rejoignez le serveur en cliquant sur « Se connecter ». Si vous rencontrez des problèmes ici, ils sont très probablement dus à un équipement audio mal configuré ou à des problèmes de réseau. Consultez la [page de dépannage du client](Client-Troubleshooting).
+
+### Le premier bœuf
+Félicitations ! Vous pouvez maintenant commencer à bœuffer avec Jamulus. Les chariots que vous voyez à droite sont votre mixage personnel. Tout ce que vous modifiez ici agit sur ce que vous entendez, mais n'affecte pas les autres utilisateurs. Si vous déplacez un chariot vers le bas, cet utilisateurs sera moins fort, si vous le déplacez vers le haut, cet utilisateur sera plus fort pour vous.
+
+![Fenêtre principale Jamulus connecté](https://user-images.githubusercontent.com/9108457/101895820-55811700-3ba8-11eb-9945-7923c8a92c60.png)
+
+Si vous ne voulez pas que les autres utilisateurs vous entendent, cliquez sur « Me silencer » (à gauche de votre mixage personnel) ce qui va empêcher d'envoyer votre signal audio aux autres musiciens. 
+
+Pendant que nous sommes ici : « Paramètres » ouvre, évidement, la fenêtre des paramètres avec tous les paramètres audio que vous voudrez probablement modifier pour vos besoins.
+
+Plus d'informations sur l'utilisation de Jamulus peuvent être trouvées dans le [manuel du logiciel](Software-Manual).
+
+## Avancé
+<!-- Bien sûr, vous pouvez en faire encore plus avec Jamulus, comme configurer un serveur privé, faire des concerts en ligne, … Jetez un coup d'œil sur ce site ! La communauté publie des guides, trucs, astuces dans la [base de connaissance](/kb/). --> 
+
+## Aide et communauté
+Si vous avez besoin d'aide ou voulez contribuer au pojet Jamulus, vous pouvez consulter les liens suivants :
+
+### Forums de discussion
+Pour de l'aide, et des discussions générales à propos de Jamulus : jetez un œil aux [forums sur Sourceforge](https://sourceforge.net/p/llcon/discussion/) (en anglais).
+
+### Développement et contribution
+Si vous voulez contribuer au logiciel Jamulus, jetez un œil au [dépot Github de Jamulus](https://github.com/corrados/jamulus/) et lisez le fichier contribution qui s'y trouve.
+
```

Comment:
I think we could put the Facebook groups here. Maybe we should open a PR to the english version. But this is to be discussed.

Reply:
Why not, and perhaps we can link to the French Bastorius website here too (for the French version).
The links are no longer in the other pages (Getting started and installtion).

In the Help and Community paragraph I think.

Reply:
Yes. I know there was a discussion somewhere about adding facebook groups etc. but I don't know where it was. Do you know the issue?

Reply:
> Do you know the issue?

No… there is a lot of discussions! ;)